websites/W/Worldhideaway/presence.ts

```diff
@@ -2,151 +2,177 @@ const presence = new Presence({
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("covers"),
+		]);
+	if (privacy) presenceData.details = "Hojeada";
+	else {
+		switch (pathname.split("/")[1]) {
+			case "": {
+				if (search?.value) {
+					presenceData.details = "Buscando";
+					presenceData.state = search.value;
+					presenceData.smallImageKey = "searching";
+				} else presenceData.details = "Viendo Inicio";
 
-			break;
-		}
-		case "/members/": {
-			presenceData.details = "Viendo:";
-			presenceData.state = "Miembros";
-
-			break;
-		}
-		case "/forums/": {
-			presenceData.details = "Viendo:";
-			presenceData.state = "Foro";
+				break;
+			}
+			case "register": {
+				presenceData.details = "Registro";
+				break;
+			}
+			case "logros": {
+				presenceData.details = "Viendo logros";
+				presenceData.buttons = [
+					{
+						label: "Viendo Logros",
+						url: href,
+					},
+				];
+				break;
+			}
+			case "events": {
+				presenceData.details = "Viendo events";
+				presenceData.buttons = [
+					{
+						label: "Viendo Events",
+						url: href,
+					},
+				];
+				break;
+			}
+			case "activity": {
+				presenceData.details = "Leyendo sobre";
+				presenceData.state = "Actividad";
+				presenceData.smallImageKey = "reading";
+				presenceData.buttons = [
+					{
+						label: "Leyendo Sobre",
+						url: href,
+					},
+				];
 
-			break;
-		}
-		default:
-			if (page.includes("/topic/")) {
-				title = document.querySelector(
+				break;
+			}
+			case "topic": {
+				presenceData.details = "Viendo noticia sobre";
+				presenceData.state = document.querySelector(
 					"#topic-6307-replies > li.bs-item-wrap.bs-header-item.align-items-center.no-hover-effect > div > div.item-title > h1"
-				);
-				presenceData.details = "Viendo Noticia Sobre:";
-				presenceData.state = title.textContent;
-			} else if (page.includes("/members/")) {
-				presenceData.details = "Viendo:";
-				presenceData.state = `${
-					document.querySelector(
-						"#item-header-content > div > div > div.flex.align-items-center.member-title-wrap > h2"
-					).textContent
-				}'s Perfil`;
-			} else if (page.includes("/category/")) {
-				title = document.querySelector("#main > header > h1 > span");
-				presenceData.details = "Viendo Categoría:";
-				presenceData.state = title.textContent;
-			} else if (page === "/faqs/") {
-				presenceData.details = "Leyendo:";
-				presenceData.state = "FAQ";
+				).textContent;
+				presenceData.buttons = [
+					{
+						label: "Viendo Noticia Sobre",
+						url: href,
+					},
+				];
+				break;
+			}
+			case "members": {
+				if (document.querySelector('[id="members-all"]')) {
+					presenceData.details = "Viendo miembros";
+					presenceData.buttons = [
+						{
+							label: "Viendo Miembros",
+							url: href,
+						},
+					];
+				} else {
+					presenceData.buttons = [
+						{
+							label: "Viendo Miembro",
```

Comment:
The label is the text that shows up on the button. Its purpose is to perform an action so you should be using the infinitive form instead of the gerund (Ver Miembro)